modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/Transaction.scala

```diff
@@ -91,7 +89,7 @@ object Transaction {
         case StorageTarget.PasswordConfig.PlainText(text) =>
           Resource.pure[F, String](text)
         case StorageTarget.PasswordConfig.EncryptedKey(StorageTarget.EncryptedConfig(key)) =>
-          Resource.eval(AWS[F].getEc2Property(key.parameterName).map(b => new String(b)))
+          Resource.eval(AWS.getEc2Property[F](key.parameterName).map(b => new String(b)))
```

Comment:
Implicit dependency between AWS and Redshift with EncryptedKey.  
It should be validated during Config parsing.

Reply:
I've created `SecretStore` interface for this purpose. GCP version will use [GCP Secret Manager](https://cloud.google.com/secret-manager) which is the counterpart of EC2 Parameter Store.